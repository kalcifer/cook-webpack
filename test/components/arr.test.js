import React from 'react';
import renderer from 'react-test-renderer';

import Arr from '../../components/arr';

test('Shows array type field', ()=>{
    let items = {'$ref':'#/definitions/externals'};
    const component = renderer.create(<Arr items={items} id={1} parentId={0}/>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})
