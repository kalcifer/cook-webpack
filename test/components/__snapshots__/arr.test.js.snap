exports[`test Shows array type field 1`] = `
<div>
  TODO - This is recursive 
</div>
`;
